Makefile

```diff
@@ -165,6 +165,16 @@ push: image
 	docker push ${REGISTRY}/yunikorn:scheduler-${VERSION}
 	docker push ${REGISTRY}/yunikorn:admission-${VERSION}
 
+#Generate the CRD code with code-generator (release-1.14)
+
+# If you want to re-run the code-generator to generate code,
+# Please make sure the directory structure must be the example.
+# ex: github.com/apache/incubator-yunikorn-k8shim
+.PHONY: code_gen
+code_gen:
+	@echo "Generating CRD code"
+	bash pkg/apis/yunikorn.apache.org/update-codegen.sh
```

Comment:
NIT: do not use bash to run executable, some Linux box may not have bash installed, we do not need to depend on that. ./path/to/script.sh is enough. 

Another thing is, can we move this script out of pkg folder? under pkg, that is expecting all .go src files. For script, I think we can create a top level folder: scripts

Reply:
If we use`./` to run sh file, the `dirname` will fail, then the output-base setting will also fail.